cli/lib/tasks/install.js

```diff
@@ -55,12 +55,15 @@ const displayCompletionMsg = () => {
 
   logger.log()
   logger.log(
-    'You can now open Cypress by running:',
-    chalk.cyan(path.join('node_modules', '.bin', 'cypress'), 'open'),
+    'You can now open Cypress by running one of the following, depending on your package manager:',
   )
 
```

Comment:
I think it'd be more readable with an extra space here:
```suggestion
logger.log()
```

Reply:
I agree with you, which means I need to regenerate the snap-shots.

Reply:
see https://github.com/cypress-io/cypress/pull/29957/commits/9219421e8ca77be0997a4e741e694bc51627ea72 for additional spacing line and snapshots